README.md

```diff
@@ -1,10 +1,12 @@
 ## Simularium repositories
 This repository is part of the Simularium project ([simularium.allencell.org](https://simularium.allencell.org)), which includes repositories:
 - [simulariumIO](https://github.com/simularium/simulariumio) - Python package that converts simulation outputs to the format consumed by the Simularium viewer website
-- [simularium-engine](https://github.com/simularium/simularium-engine) - C++ backend application that interfaces with biological simulation engines and serves simulation data to the front end website
+- [octopus](https://github.com/simularium/octopus) - Python application that serves simulation data to the front end website
 - [simularium-viewer](https://github.com/simularium/simularium-viewer) - NPM package to view Simularium trajectories in 3D
 - [simularium-website](https://github.com/simularium/simularium-website) - Front end website for the Simularium project, includes the Simularium viewer
 
+## Simularium Project Status
+While we are continuing to support Simularium, **active development on new features is currently paused**. As a result, new feature request will not be addressed immediately.
```

Comment:
I'm not totally sure of the benefit of doing this. Do you want to change all the simularium readmes in this way?